helpers/session.go

```diff
@@ -0,0 +1,82 @@
+package helpers
+
+import (
+	"go-google-scraper-challenge/helpers/log"
+	"go-google-scraper-challenge/lib/models"
+
+	"github.com/gin-contrib/sessions"
+	"github.com/gin-gonic/gin"
+)
+
+const CurrentUserKey = "CURRENT_USER_ID"
+const (
+	FlashTypeSuccess = "success"
+	FlashTypeInfo    = "info"
+	FlashTypeError   = "error"
+)
+
+type Flash struct {
+	Type    string
+	Message string
+}
+
+func GetCurrentUser(ctx *gin.Context) *models.User {
+	session := sessions.Default(ctx)
+	currentUserID := session.Get(CurrentUserKey)
+	if currentUserID == nil {
+		return nil
+	}
+
+	user, err := models.GetUserByID(currentUserID.(int64))
+	if err != nil {
+		log.Error("Fail to get current user", err.Error())
+
+		return nil
+	}
+
+	return user
+}
+
+func SetCurrentUser(ctx *gin.Context, userID int64) {
+	session := sessions.Default(ctx)
+	session.Set(CurrentUserKey, userID)
+	err := session.Save()
```

Comment:
I'm unsure which formatting you're going to use here as I see that in the next function you have a space before this section 🫣

Reply:
I removed the space ad there's no guard or return statement 😅 [1666742](https://github.com/carryall/go-google-scraper-challenge/pull/122/commits/1666742f83a3d1d0abaee345d12ea739e8237f34)